src/checkFirebaseSDKVersion.ts

```diff
@@ -0,0 +1,92 @@
+"use strict";
+
+import * as _ from "lodash";
+import * as clc from "cli-color";
+import * as path from "path";
+import * as semver from "semver";
+import { spawnSync } from "child_process";
```

Comment:
I saw this change and figured I'd bring it up now (don't know if this is ready for review yet): `cross-spawn` is used so that processes can be spun up in Windows and Linux environments. The built-in handlers were having trouble with this.

Reply:
This PR is still in draft, but good to know thanks!

Reply:
Changed back to spawnSync